src/Checkerboard/CheckerboardLayout.vala

```diff
@@ -1079,15 +1076,43 @@ public class CheckerboardLayout : Gtk.DrawingArea {
         // find the visible intersection of the viewport and the selection band
         Gdk.Rectangle visible_page = get_adjustment_page (hadjustment, vadjustment);
         Gdk.Rectangle visible_band = Gdk.Rectangle ();
+
         visible_page.intersect (selection_band, out visible_band);
-        ctx.rectangle (visible_band.x, visible_band.y, visible_band.width, visible_band.height);
-        var style_context = get_style_context ();
-        style_context.save ();
-        style_context.add_class (Granite.STYLE_CLASS_ACCENT);
-        var selected_rgba = style_context.get_color (Gtk.StateFlags.NORMAL);
-        ctx.set_source_rgba ( selected_rgba.red, selected_rgba.green, selected_rgba.blue, SELECTION_ALPHA);
+
+        var label_widget_path = new Gtk.WidgetPath ();
+        label_widget_path.append_type (typeof (Gtk.IconView));
+        label_widget_path.iter_set_object_name (-1, "rubberband");
+
+        var rubberband_context = new Gtk.StyleContext ();
+        rubberband_context.set_path (label_widget_path);
+
+        var bg_color = (Gdk.RGBA) rubberband_context.get_property (
+            Gtk.STYLE_PROPERTY_BACKGROUND_COLOR,
+            Gtk.StateFlags.NORMAL
+        );
+
```

Comment:
`StyleContext.get_background_color ()` is deprecated and this seems to do the same thing (although it is not currently deprecated for some reason). You are supposed to use StyleContext.render_background () which should also render the appropriate border and corners.  I'll see whether this will work with this constructed style context.